src/main/java/edu/iu/terracotta/controller/app/AssignmentController.java

```diff
@@ -170,4 +171,32 @@ public ResponseEntity<Void> deleteAssignment(@PathVariable("experiment_id") Long
             return new ResponseEntity(TextConstants.NOT_ENOUGH_PERMISSIONS, HttpStatus.UNAUTHORIZED);
         }
     }
+
+    @Transactional
+    @PostMapping(value = "/{experimentId}/exposures/{exposureId}/assignments/{assignmentId}/duplicate")
+    public ResponseEntity<AssignmentDto> postAssignment(@PathVariable long experimentId,
```

Comment:
Can we name this method `duplicateAssignment`?

Reply:
Done! Good catch!